core/src/main/scala/scalaz/MonadTrans.scala

```diff
@@ -15,6 +15,9 @@ trait MonadTrans[F[_[_], _]] {
   /** The [[scalaz.Monad]] implied by this transformer. */
   implicit def apply[G[_] : Monad]: Monad[F[G, ?]]
 
+  def mapF[G[_], A, B](fa: F[G, A])(f: A => G[B])(implicit M: Monad[G]): F[G, B] =
+    Monad[F[G, ?]].bind(fa)(a => liftM(f(a)))
+
```

Comment:
@TomasMikula I think this is true, it is confusing.  Seems we would break backwards compatibility either way if we change `mapF` to `mapM` or `liftM` to `liftF`?  I don't have strong feelings.  @xuwei-k ?